### DISCUSSION TITLE
Just turned on SSO and now we can't push

### DISCUSSION THREAD

**Discussion by fm456** [community/community#30991] [Code Security]
I get the following error when trying to push a new branch into my repo (this worked just fine before enabling SSO)
"To access
this repository, you must use the HTTPS remote with a personal access token
or SSH with an SSH key and passphrase
that has been authorized for this organization. Visit
https://docs.github.com/articles/authenticating-to-a-github-organization-with-saml-single-sign-on/ for more information.

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists."

I visit the link and the documentation makes sense I need to enable my ssh key for use with SSO.  I go to my ssh key and I don't have the drop down to let me enable SSO access.  The SSO test button worked fine and SSO seems to be working as it is supposed to be. I have tried the easy things like logging out and back in, recreating the ssh key, and revoking my SSO identity all of which didn't change anything.  Any help would be appreciated.

**Comment by GwenVCX** [comment]
Setting up access this way has worked for me https://www.youtube.com/watch?v=hPmxmpNbcH8
